lib/pages/edit_profile_page.dart

```diff
@@ -245,20 +280,20 @@ class  EditProfileFormState  extends State<EditProfileForm> {
       final FirebaseAuth auth = FirebaseAuth.instance;
       final currentUser = auth.currentUser;
       if (currentUser != null) {
-        if (username != "") {
+        if (username.isNotEmpty) {
           final data = { "username.profile_name" :username};
           await db.collection("profile_data").doc(currentUser.uid).update(data);
         }
-        if(firstname != "")
+        if(firstname.isEmpty)
```

Comment:
@shan-226 I believe this should be isNotEmpty?

Reply:
Oops, thank you for spotting this. Fixed.